src/Pages/Search/SearchResult.js

```diff
@@ -63,7 +70,7 @@ const SearchResult = () => {
           }}
         >
           {searchedRecipe.length ? (
-            searchedRecipe?.map((item) => {
+            pageData.currentData().map((item) => {
               return (
```

Comment:
We need to make some changes here to 75
<Link to={"/recipe/" + recipe.id} key={recipe.id}>
                      <ReusableCard
                        key={recipe.id}
                        title={recipe.title}
                        data={recipe}
                        image={recipe.image}
                      />
                    </Link> 
Because when I click on the recipe, it doesn't open detailed info about the recipe.

Reply:
done. Review again please